### PR TITLE
FE 201: Styling Essentials: CSS Cross Country

### PR DESCRIPTION
**Pull request by nicoRDL**
## Description

- creates new homepage named index
- adds styling for new file
- updates existing pages to include a category tag
- updates page linking and interaction

## Related Ticket

https://codeandtheory.atlassian.net/browse/FE-650

## Testing Instructions

1. Pull on FE-201/FE-650 branch
2. evaluate output

## Risks
N/A

### CODE REVIEW

File: view/detail-page-light-sample1.html [hunk 1]
```diff
@@ -19,7 +19,8 @@
         alt="colored keyboard logo"
       />
       <ul>
-        <li><a href="/view/browse-page.html">Home</a></li>
+        <li><a href="/view/index.html">Home</a></li>
+        <li><a href="/view/browse-page.html">Browse</a></li>
```
Comment (lycoangelo) [inline view/detail-page-light-sample1.html]: Please update the `href` values. These are broken links.

File: view/detail-page-light-sample1.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
   </head>
   <body>
     <nav class="nav">
-      <a class="title-logo" href="/view/browse-page.html">
+      <a class="title-logo" href="/view/index.html">
```
Comment (lycoangelo) [inline view/detail-page-light-sample1.html]: Please update the `href` values. These are broken links.

File: view/detail-page-light-sample2.html [hunk 3]
```diff
@@ -9,7 +9,7 @@
   </head>
   <body>
     <nav class="nav">
-      <a class="title-logo" href="/view/browse-page.html">
+      <a class="title-logo" href="/view/index.html">
```
Comment (lycoangelo) [inline view/detail-page-light-sample2.html]: Please update the `href` values. These are broken links.

File: view/detail-page-light-sample2.html [hunk 4]
```diff
@@ -19,7 +19,8 @@
         alt="colored keyboard logo"
       />
       <ul>
-        <li><a href="/view/browse-page.html">Home</a></li>
+        <li><a href="/view/index.html">Home</a></li>
+        <li><a href="/view/browse-page.html">Browse</a></li>
```
Comment (lycoangelo) [inline view/detail-page-light-sample2.html]: Please update the `href` values. These are broken links.

File: view/detail-page-sample1.html [hunk 5]
```diff
@@ -9,7 +9,7 @@
   </head>
   <body>
     <nav class="nav">
-      <a class="title-logo" href="/view/browse-page.html">
+      <a class="title-logo" href="/view/index.html">
```
Comment (lycoangelo) [inline view/detail-page-sample1.html]: Please update the `href` values. These are broken links.

File: view/detail-page-sample2.html [hunk 6]
```diff
@@ -9,7 +9,7 @@
   </head>
   <body>
     <nav class="nav">
-      <a class="title-logo" href="/view/browse-page.html">
+      <a class="title-logo" href="/view/index.html">
```
Comment (lycoangelo) [inline view/detail-page-sample2.html]: Please update the `href` values. These are broken links.

File: view/detail-page-sample2.html [hunk 7]
```diff
@@ -19,7 +19,8 @@
         alt="colored keyboard logo"
       />
       <ul>
-        <li><a href="/view/browse-page.html">Home</a></li>
+        <li><a href="/view/index.html">Home</a></li>
+        <li><a href="/view/browse-page.html">Browse</a></li>
```
Comment (lycoangelo) [inline view/detail-page-sample2.html]: Please update the `href` values. These are broken links.

File: view/detail-page-sample3.html [hunk 8]
```diff
@@ -9,7 +9,7 @@
   </head>
   <body>
     <nav class="nav">
-      <a class="title-logo" href="/view/browse-page.html">
+      <a class="title-logo" href="/view/index.html">
```
Comment (lycoangelo) [inline view/detail-page-sample3.html]: Please update the `href` values. These are broken links.

File: view/detail-page-sample3.html [hunk 9]
```diff
@@ -19,7 +19,8 @@
         alt="colored keyboard logo"
       />
       <ul>
-        <li><a href="/browse-page.html">Home</a></li>
+        <li><a href="/view/index.html">Home</a></li>
+        <li><a href="/view/browse-page.html">Browse</a></li>
```
Comment (lycoangelo) [inline view/detail-page-sample3.html]: Please update the `href` values. These are broken links.

File: view/index.html [hunk 10]
```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/css/main.css" />
+    <link rel="stylesheet" href="/css/browse-page.css" />
+    <link rel="stylesheet" href="/css/home-page.css" />
```
Comment (lycoangelo) [inline view/index.html]: Please update the `href` values. These are broken links.

File: view/index.html [hunk 11]
```diff
@@ -0,0 +1,196 @@
...
+        <img
+          class="nav-logo"
+          src="/images/keyboard-logo.png"
+          alt="colored keyboard logo"
+        />
+        <h2>Clacks&Thocc</h2>
+      </a>
+      <ul>
+        <li><a href="/view/index.html">Home</a></li>
+        <li><a href="/view/browse-page.html">Browse</a></li>
```
Comment (lycoangelo) [inline view/index.html]: Please update the `href` values. These are broken links.

File: view/article-page.html [hunk 12]
```diff
@@ -11,7 +11,7 @@
   </head>
   <body>
     <nav class="nav">
-      <a class="title-logo" href="/view/browse-page.html">
+      <a class="title-logo" href="/view/index.html">
```
Comment (lycoangelo) [inline view/article-page.html]: Please update the `href` values. These are broken links.

File: view/article-page.html [hunk 13]
```diff
@@ -21,7 +21,8 @@
         alt="colored keyboard logo"
       />
       <ul>
-        <li><a href="/view/browse-page.html">Home</a></li>
+        <li><a href="/view/index.html">Home</a></li>
+        <li><a href="/view/browse-page.html">Browse</a></li>
```
Comment (lycoangelo) [inline view/article-page.html]: Please update the `href` values. These are broken links.

File: view/browse-page.html [hunk 14]
```diff
@@ -10,7 +10,7 @@
   </head>
   <body>
     <nav class="nav">
-      <a class="title-logo" href="/view/article-page.html">
+      <a class="title-logo" href="/view/index.html">
```
Comment (lycoangelo) [inline view/browse-page.html]: Please update the `href` values. These are broken links.

File: view/browse-page.html [hunk 15]
```diff
@@ -20,7 +20,7 @@
         alt="colored keyboard logo"
       />
       <ul>
-        <li><a href="/view/browse-page.html">Home</a></li>
+        <li><a href="/view/index.html">Home</a></li>
```
Comment (lycoangelo) [inline view/browse-page.html]: Please update the `href` values. These are broken links.

File: view/detail-page-sample1.html [hunk 16]
```diff
@@ -19,7 +19,8 @@
         alt="colored keyboard logo"
       />
       <ul>
-        <li><a href="/view/browse-page.html">Home</a></li>
+        <li><a href="/view/index.html">Home</a></li>
+        <li><a href="/view/browse-page.html">Browse</a></li>
```
Comment (lycoangelo) [inline view/detail-page-sample1.html]: Please update the `href` values. These are broken links.

File: view/index.html [hunk 17]
```diff
@@ -0,0 +1,196 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/css/main.css" />
+    <link rel="stylesheet" href="/css/browse-page.css" />
+    <link rel="stylesheet" href="/css/home-page.css" />
+    <title>Clacks & Thocc - Home Page</title>
+  </head>
+  <body>
+    <nav class="nav">
+      <a class="title-logo" href="/view/index.html">
```
Comment (lycoangelo) [inline view/index.html]: Please update the `href` values. These are broken links.

**Review by lycoangelo — CHANGES_REQUESTED** [review]

**Comment by nicoRDL** [comment]
@lycoangelo updated PR to fix broken links and file paths. Requesting for re-review 

https://github.com/nicoRDL/dotTheory-FEv2/pull/3/commits

**Review by lycoangelo — APPROVED** [review]

**Pull request merged**